plugins/source/aws/policies/queries/ec2/security_groups_with_open_critical_ports.sql

```diff
@@ -36,5 +33,20 @@ select
         or 9300 between from_port and to_port)
         then 'fail'
         else 'pass'
-    end
-from view_aws_security_group_ingress_rules
+    end as status
+  FROM view_aws_security_group_ingress_rules
+)
+
+SELECT
+	:'execution_time' as execution_time,
+	:'framework' as framework,
+	:'check_id' as check_id,
+	'Aggregates rules of security groups with ports and IPs including ipv6' as title,	
+    account_id,
+    resource_id,
+    CASE
+      WHEN SUM(CASE WHEN status = 'fail' THEN 1 ELSE 0 END) > 0 THEN 'fail'
```

Comment:
Approving this for now - let's choose a consistent way across our logic to do aggregate checks like this.